### PR TITLE
chore: bump sip-010 trait to its latest version

### LINKED ISSUE #22: SIP 010 is out of date
The latest SIP 010 trait includes a token memo.

### PR DESCRIPTION
**Pull request by LNow**
This PR bumps SIP-010 trait to its latest version.

close #22

### CODE REVIEW

File: contracts/citycoin.clar [hunk 1]
```diff
@@ -854,13 +854,9 @@
 )
 
 ;;;;;;;;;;;;;;;;;;;;; SIP 010 ;;;;;;;;;;;;;;;;;;;;;;
-;; name change: per https://github.com/stacksgov/sips/pull/25
-;; .sip-10-ft-standard.sip-010-trait
-(impl-trait 'SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.sip-10-ft-standard.ft-trait)
+(impl-trait 'SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.sip-010-trait.sip-010-trait)
```
Comment (whoabuddy) [inline contracts/citycoin.clar:857]: Can you add a comment with the testnet version here as well so its not forgotten / easier to find when deploying to testnet?

```lisp
;; testnet implementation
;; (impl-trait 'ST1HTBVD3JG9C05J7HBJTHGR0GGW7KXW28M5JS8QE.sip-010-trait.sip-010-trait)
```

**Review by whoabuddy — CHANGES_REQUESTED** [review]
One itty-bitty little tiny change and LGTM!

File: contracts/citycoin.clar (hunk 1 above)
Comment (LNow) [inline contracts/citycoin.clar:857]:
```suggestion
;; testnet implementation
;; (impl-trait 'ST1HTBVD3JG9C05J7HBJTHGR0GGW7KXW28M5JS8QE.sip-010-trait.sip-010-trait)
(impl-trait 'SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.sip-010-trait.sip-010-trait)
```

Comment (LNow) [reply contracts/citycoin.clar:859]: Done

**Review by whoabuddy — APPROVED** [review]

**Review by asteriabtc — APPROVED** [review]

**Pull request merged**